docs/arcaflow/workflows/flow-control.md

```diff
@@ -199,90 +217,15 @@ iterations:
 
 To use the generated values from `bindConstants()`, a new schema representing these bound values must be added to the input schema section of our `subworkflow.yaml`, `input`. This new schema's ID will be the ID of the schema that defines the items in your list, in this case `SubRootObject` and the schema name that defines your repeated inputs, in this case `RepeatedValues`, concatenated with a double underscore, `__`. This creates our new schema ID, `SubRootObject__RepeatedValues`. You are required to use this schema ID because it is generated from the names of your other schemas.
 
-```yaml title="workflow.yaml"
-version: v0.2.0
-input:
-  root: RootObject
-  objects:
-    RootObject:
-      id: RootObject
-      properties:
-        repeated_inputs:
-          type:
-            type_id: ref
-            id: RepeatedValues
-        iterations:
-          type:
-            type_id: list
-            items:
-              id: SubRootObject
-              type_id: ref
-    RepeatedValues:
-      id: RepeatedValues
-      properties:
-        name:
-          type:
-            type_id: string
-    SubRootObject:
-      id: SubRootObject
-      properties:
-        loop_id:
-          type:
-            type_id: integer            
-            
+```yaml title="workflow.yaml"            
 steps:
   foreach_loop:
     kind: foreach
     items: !expr 'bindConstants($.input.iterations, $.input.repeated_inputs)'
     workflow: subworkflow.yaml
     parallelism: 1
-
-outputs:
-  success:
-    fab_four: !expr $.steps.foreach_loop.outputs.success.data
 ```
 
-```yaml title="subworkflow.yaml"
-version: v0.2.0
-input:
-  root: SubRootObject__RepeatedValues
-  objects:
-    SubRootObject__RepeatedValues:
-      id: SubRootObject__RepeatedValues
-      properties:
-        constant:
-          type:
-            type_id: ref
-            id: RepeatedValues
-        item:
-          type:
-            type_id: ref
-            id: SubRootObject
-    RepeatedValues:
-      id: RepeatedValues
-      properties:
-        name:
-          type:
-            type_id: string
-    SubRootObject:
-      id: SubRootObject
-      properties:
-        loop_id:
-          type:
-            type_id: integer
-
-steps:
-  example:
-    plugin:
-      deployment_type: image
-      src: quay.io/arcalot/arcaflow-plugin-template-python:all_d5dd9d6
-    input:
-      name: !expr $.input.constant.name
-
-outputs:
-  success:
-    loop_id: !expr $.input.item.loop_id
-    beatle: !expr $.steps.example.outputs.success
-```
+See the [full workflow](www.redhat.com).
```

Comment:
That URL is just a placeholder, I take it, until https://github.com/arcalot/arcaflow-workflows/pull/31 is merged?  (This comment is just a placeholder, to remind you to replace the placeholder....  🙂)

Reply:
changed